schemas/publication.json

```diff
@@ -59,12 +59,14 @@
 			"description": "Optional metadata that contains an array of references for works cited in this submission. References have been split and parsed from the raw text.",
 			"readOnly": true,
 			"items": {
-				"type": "string"
+				"type": "object",
+				"$ref": "#/definitions/Citation"
 			}
 		},
-		"citationsRaw": {
+		"rawCitations": {
```

Comment:
I believe it was called "citationsRaw" in order to be direct behind "citations", because the properties are alphabetically ordered. 
So maybe to leave it as it was i.e. "citationsRaw"